packages/experiments/src/components/Recommendation/Recommendation.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+
+/* Dependent Components */
+import { PrimaryButton } from 'office-ui-fabric-react/lib/Button';
+import { CardFrame, ICardDropDownOption } from '../Card/CardFrame';
```

Comment:
>../Card/CardFrame [](start = 48, length = 17)

Why is this relative path? Shouldn't we be getting this from office-ui-fabric-react, or is it because you are getting this from experimental as well?

Reply:
Yes i agree , but the way Jest snapshot tests work, they need a relative path. If we provide a module path they fail to resolve the module, looking at other components with snapshot tests they follow similar path too.

Reply:
Also, @ uifabric/experiements/Card/CardFrame is basically an alias in webpack.config for the relative path rootDir/lib/Card/CardFrame. Then i need to modify jest.config to look for this path whenever it encounters that module, that's another way to resolve this.

Reply:
I was initially hesitant on fixing jest.config (unsure what other tests might get impacted), went ahead and modified to be able to understand the module. See the change here 'packages/experiments/jest.config.js' , ran all tests looks to be working fine. Removed relative path.

---
In reply to: [196484495](https://github.com/OfficeDev/office-ui-fabric-react/pull/5252#discussion_r196484495) [](ancestors = 196484495)